pkg/odo/cli/dev/dev.go

```diff
@@ -28,7 +34,7 @@ import (
 // RecommendedCommandName is the recommended command name
 const RecommendedCommandName = "dev"
 
-type DevOptions struct {
+type Options struct {
```

Comment:
Why renaming it like this? It is not following the "implicit" convention

Reply:
The IDE suggested to do so because it's inside dev package and the word "dev" was repeating. For example - `dev.DevOptions` vs. `dev.Options`.

Reply:
OK. I'll remove it for now.